src/main.cpp

```diff
@@ -20,7 +20,7 @@ int main(int argc, char* argv[])
 
   bool WarmStartingFlag = false;
   int Resolution = 0;
-  double nu1 = 0.0, nu2 = 0.0, CompositeYoungs = 0.0, alpha = 0.0,
+  double nu1 = 0.0, nu2 = 0.0, CompositeYoungs = 0.0, CompositePoissonsRatio = 0.0, alpha = 0.0,

```

Comment:
What is `CompositePoissonsRatio`? Also, is there a documentation of all these parameters somewhere?

Reply:
You can look into Bonari (2020): https://doi.org/10.1007/s00466-019-01791-3
See equations 14a and 14b (page 736).

Reply:
The approach from Bonari2020 should already be implemented, why is this now added for the new pressure type version?

Reply:
The influence coefficients in the force-based Green function did not require the calculation of composite Poisson's ratio, but only the calculation of composite Young's modulus. However, the pressure-based Green function requires both to calculate the influence coefficients.